README.md

```diff
@@ -1,42 +1,34 @@
 Using the repository template
 =============================
 
-
-
 #### Pre-requisites:
  - Mac OS or Linux OS
+ - [Python 2.X](https://www.python.org)
  - [Stata MP](http://www.stata.com/statamp/)
-    - Executable should also be added to the `PATH` environment variable so it is callable from the command line with `statamp`
  - [R](https://www.r-project.org/)
-    - Executable should also be added to the `PATH` environment variable so it is callable from the command line with `Rscript`
- - [scons](http://scons.org/) 
  - [Lyx](https://www.lyx.org/)
-    - Executable should also be added to the `PATH` environment variable so it is callable from the command line with `lyx`
+    - For the above four, executables need to added to `path` environment variable. So they are callable from the command line with `python`, `statamp`, `Rscript`, and `lyx`.
+ - [SCons](http://scons.org/) version 2.4.0 or later
+    - SCons can also be installed via [pip](https://pip.pypa.io/en/stable/) using `pip install --egg scons`.
+    - More information about Scons can be found [here](https://github.com/gslab-econ/ra-manual/wiki/SCons).
  - [git-lfs](https://git-lfs.github.com/)
+ - [gslab_tools](https://pypi.python.org/pypi/GSLab_Tools) version 1.1.1 or later
+    - Installed via [pip](https://pip.pypa.io/en/stable/) using `pip install gslab_tools`.
+ - [gslab_stata tools](https://github.com/gslab-econ/gslab_stata)
+    - Installed according to [computer-build-sheet](https://github.com/gslab-econ/admin/blob/cd4ddcf91cbd724a0ac444adf18bdba797457e3c/computer-build-sheet/package_build.sh).
```

Comment:
@lboxell if it is easy to do I would support removing the dependency on matrix_to_txt since that's not needed to illustrate the functionality of the template.